pkg/feature/raw_resources.go

```diff
@@ -106,7 +97,7 @@ func (f *Feature) patchResourceFromFile(filename string) error {
 		}
 
 		// Convert the patch from YAML to JSON
-		patchAsJSON, err := yaml.YAMLToJSON(data)
+		patchAsJSON, err := yaml.YAMLToJSON([]byte(data))
```

Comment:
I think there's a bug in here. We split the whole thing into chunks based on `---` separate, but instead of patching resources one by one we apply the entire `data` (so all the resources all at once). This means that currently we only have single-resource files in use, that's why this bug never showed up.

We should either give up on the assumption there can be more than one resource per file OR fix it to apply one by one. I would vouch for fixing it. Can you take care of it in this PR?

Reply:
[26c0da8](https://github.com/maistra/opendatahub-operator/pull/15/commits/26c0da8a07f4608a6e2cfd3588529f44e3f2d251)